### PR TITLE
Add subcommands for right sizing ecs clusters and invoking lambda functions

### PR DESCRIPTION
**Pull request by fillup**

### CODE REVIEW

File: lib/ecs.go [hunk 1]
```diff
@@ -155,11 +166,118 @@ func GetPendingTasksCount(awsSess *session.Session, cluster string) int64 {
...
+func GetMemoryCpuNeededForEcsServices(awsSess *session.Session, ecsServices []*ecs.Service) (int64, int64) {
+	var memoryNeeded int64 = 0
+	var cpuNeeded int64 = 0
+	var largestServiceMemory int64 = 0
+	var largestServiceCpu int64 = 0
+
+	svc := ecs.New(awsSess)
+
+	for _, service := range ecsServices {
+		if *service.DesiredCount > 0 {
```
Comment (Baggerone) [inline lib/ecs.go]: How about avoiding the extra indentations with ...
```
  if *service.DesiredCount == 0 {
    continue
  }

  taskDef, err := ...
```

File: lib/ecs.go [hunk 2]
```diff
@@ -155,11 +166,118 @@ func GetPendingTasksCount(awsSess *session.Session, cluster string) int64 {
...
+	if largestDesiredCount > serversNeeded && atLeastServiceDesiredCount {
+		serversNeeded = largestDesiredCount
+	}
+
+	asgDesired, asgMin, asgMax := GetAsgServerCount(awsSess, asgName)
+	fmt.Printf("ASG server count currently set to: desired = %v, min = %v, max = %v\n", asgDesired, asgMin, asgMax)
+
+	if asgMin < serversNeeded {
+		fmt.Printf("ASG needs to be scaled up by %v servers\n", serversNeeded-asgMin)
+		fmt.Printf("Scaling ASG to %v servers (desired/min/max)...", serversNeeded)
```
Comment (Baggerone) [inline lib/ecs.go]: Do you really want `(desired/min/max)`  in these print statements?  (here and below)

File: lib/ecs.go [hunk 3]
```diff
@@ -155,11 +166,118 @@ func GetPendingTasksCount(awsSess *session.Session, cluster string) int64 {
...
+	}
+
+	return nil
+}
+
+func GetLargestDesiredCountFromEcsServices(ecsServices []*ecs.Service) int64 {
+	largestDesiredCount := int64(0)
+
+	for _, service := range ecsServices {
+		if *service.DesiredCount > 0 {
```
Comment (Baggerone) [inline lib/ecs.go]: Should this be 
```
    if *service.DesiredCount > largestDesiredCount {
        ...
```

Comment (fillup) [reply lib/ecs.go]: yes, good catch

File: lib/ecs.go (hunk 2 above)
Comment (fillup) [reply lib/ecs.go]: fixed

File: lib/ecs.go (hunk 1 above)
Comment (fillup) [reply lib/ecs.go]: fixed

File: cmd/lambdaInvoke.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+)
+
+var functionName string
+var payload string
+
+// invokeCmd represents the invoke command
+var invokeCmd = &cobra.Command{
+	Use:   "invoke",
+	Short: "Invoke a lambda function",
+	Long: `A longer description that spans multiple lines and likely contains examples
```
[the remark this answers is not in the document]
Comment (fillup) [reply cmd/lambdaInvoke.go]: fixed

**Pull request merged**